web/src/user_group_pill.ts

```diff
@@ -47,32 +59,36 @@ export function get_user_ids(pill_widget) {
     return user_ids;
 }
 
-export function append_user_group(group, pill_widget) {
+export function append_user_group(group: UserGroup, pill_widget: UserGroupPillWidget): void {
     if (group !== undefined && group !== null) {
```

Comment:
This seems to be an overly defensive check.
Because we haven't implemented the UI for user groups, it is safe to remove this check now without worrying about breaking things.